Makefile

```
@@ -0,0 +1,147 @@
+.PHONY: help build test docker pack push clean run-pathfinder run-rpc run-postgres test-rpc test-rpc-prod test-rpc-regression docker-up docker-down docker-logs call-rpc
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)